### PR TITLE
Count fields in DEX

### PR DESCRIPTION
**Pull request by jaredrummler**
The field count can surpass the 65k limit too. This adds the argument "--count-fields" to generate the field count instead of the method count on DEX.

### CODE REVIEW

File: src/info/persistent/dex/DexFieldCounts.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                Node packageNode = packageTree;
+                for (int i = 0; i < packageNamePieces.length && i < maxDepth; i++) {
+                    packageNode.count++;
+                    String name = packageNamePieces[i];
+                    if (packageNode.children.containsKey(name)) {
+                        packageNode = packageNode.children.get(name);
+                    } else {
+                        Node childPackageNode = new Node();
+                        if (name.length() == 0) {
+                            // This method is declared in a class that is part of the default package.
```
Comment (mihaip) [inline src/info/persistent/dex/DexFieldCounts.java]: Is this comment still relevant for fields?

Comment (jaredrummler) [inline src/info/persistent/dex/DexFieldCounts.java]: s/method/field/g

Not sure if this is necessary for fields.

Comment (jaredrummler) [inline src/info/persistent/dex/DexFieldCounts.java]: Would you like me to change the comment, close this PR, and create another PR? Is getting the field count out of the scope of this project?

**Comment by mihaip** [comment]
I'll fix up the comment, thanks for the PR!

**Pull request merged**